src/twisted/mail/pop3.py

```diff
@@ -360,7 +364,9 @@ def formatUIDListLines(msgs, getUidl):
     for i, m in enumerate(msgs):
         if m is not None:
             uid = getUidl(i)
-            yield '%d %s\r\n' % (i + 1, uid)
+            if not isinstance(uid, bytes):
+                uid = str(uid).encode("utf-8")
+            yield intToBytes(i + 1) + b' ' + uid + b'\r\n'
```

Comment:
I *think* if `uid` consists of only ascii chars, lines 367 and 368 can be replaced with `networkString`. But it's not entirely clear from the docs, it says "printable characters". Do have more info on this?

Reply:
I'm moving away from `networkString`